package.json

```diff
@@ -23,7 +23,7 @@
     "@babel/cli": "^7.8.4",
     "@babel/core": "^7.9.6",
     "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.9.6",
+    "@babel/plugin-transform-runtime": "^7.19.1",
```

Comment:
Are you sure this should be changed? Is it needed? Also files like package-lock.json or yarn.lock should never be pushed to repo.